tailscale/DOCS.md

```diff
@@ -347,6 +347,19 @@ CGNAT networks). You can test connections with `tailscale ping
 
 When not set, an automatically selected port is used by default.
 
+## Healthcheck
+
+Tailscale is quite resilient and can recover from nearly any network change. In
+case it fails to recover, the add-on's health is set unhealthy. The add-on's
+health is checked by Home Assistant in each 30s, and if it reports itself 3
+times unhealthy in a row, the add-on will be restarted.
+
+The add-on's health is set unhealthy:
```

Comment:
Healthy or not, and even the healthcheck itself, is an internal; not user-facing.

Explaining this to an end-user seems wrong, as they have no clue what an "health check" even remotely it.

This part of the docs would thus be fully up in the air for most readers.